Automattic/jetpack
# Related Posts: remove duplicated HTML attributes

## Proposed changes:

This PR fixes a Related Posts e2e test failing in Calypso (see the conversation linked below for more detail).

The test failed because the selector `aria-label="Block: Related Posts"` returned two elements. In this PR, we remove a DOM node that duplicates attributes already present on another node, including the `aria-label` mentioned before. It wasn't of any use for the editor block.

### Other information:


## Jetpack product discussion


See conversation: p1707859393377499-slack-C034JEXD1RD

## Does this pull request change what data or activity we track or use?


n/a

## Testing instructions:






_Warning: don't check out this branch just yet._

### Reproducing the issue

- Spin up a test site that runs the `trunk` branch
- Create a new post
- Add the Related Posts block
- Open the dev console
- In the markup, search for: `Block: Related Posts`
- You should find two occurrences
<img width="503" alt="Screenshot 2024-02-14 at 11 33 19 AM" src="https://github.com/Automattic/jetpack/assets/1620183/e58b77db-c9c1-433e-9cce-594e5e1d23c2">


### Testing the fix

- Using the same test site, check out this branch
- Rebuild the Jetpack plugin if you're testing locally
- Visit the post you've created
- Check it's not broken
- Verify you find only one instance of `Block: Related Posts` in the markup
